docs/topics/development/data_management.md

```diff
@@ -35,30 +35,47 @@ The `make initialize` command, executed as part of `make initialize_docker`, per
 
 ## Exporting and Loading Data Snapshots
 
-You can export and load data snapshots to manage data states across different environments or for backup purposes. The Makefile provides commands to facilitate this.
+You can export and load data snapshots to manage data states across different environments or for backup purposes. The Makefile provides commands to facilitate this. These commands rely internally on django `loaddata` and `dumpdata` commands
+with specific logic to save/load backups in our specified backup directory.
 
-- **Exporting Data**:
+- **Data dump**:
 
   ```sh
-  make data_export [EXPORT_DIR=<path>]
+  make data_dump [ARGS="--name <name> --force"]
   ```
 
-  This command creates a dump of the current MySQL database. The optional `EXPORT_DIR` argument allows you to specify a custom path for the export directory.
-  The default value is a timestamp in the `backups` directory.
+  This command creates a dump of the current MySQL database. The command accepts an optional `name` argument which will determine
+  the name of the directory created in the `DATA_BACKUP_DIR` directory. By default it uses a timestamp to ensure uniqueness.
 
-  The data exported will be a .sql dump of the current state of the database including any data that has been added or modified.
+  You can also specify the `--force` argument to overwrite an existing backup with the same name.
 
 - **Loading Data**:
 
   ```sh
-  make data_restore [RESTORE_DIR=<path>]
+  make data_load [ARGS="--name <name>"]
   ```
 
-  This command restores a MySQL database from a previously exported snapshot. The optional `RESTORE_DIR` argument allows you to specify the path of the import file.
-  This must be an absolute path. It defaults to the latest stored snapshot in the `backups` directory.
+  This command will load data from an existing backup directory. The name is required and must match a directory in the `DATA_BACKUP_DIR` directory.
 
-Refer to the Makefile for detailed instructions on these commands.
+  > NOTE: This command will NOT reindex elasticsearch. In most cases you should use the `make initialize_data` command instead.
```

Comment:
This will be handled in a follow up PR where we introduce the `initialize` command. This will handle indexing. I wouldn't want to introduce that concept here as the load/dump commands are really just about managing the data and (for now at least) I do not know of a good way to dump/load the es index.

Reply:
This is about `data_load` though - that can be used at any time to load data from your backup folder -  not just during the first run `initialize`.  `make restore` used to call reindex so what's the argument against doing it for `make data_load`?

Reply:
I wouldn't worry about trying to dump/load ES and would straight up reindex (with `--wipe` if you want to ensure a clean slate).

I agree with @eviljeff that if you're loading data you should reindex to keep things consistent.

Reply:
I don't want to make `data_load` the command you actually run.

If you want to run with a specific backup you will actually be able to run

```sh
make up INIT_LOAD=<backup_name>
```

Adding all of the functionality everywhere just adds complexity, more testing (can you opt out of indexing on data_load)... data_load is about "just" loading data. that's it.

Reply:
You _could_ make the argument that reindex **only** get's called during `data_load` and then it would be totally internalized.

We would have to remove the ability to opt out of indexing though.. as it would be coupled to loading. IDK I can see both sides. @diox do you have an opinion?

Reply:
I'm coming around to your viewpoint actually. `load_data` totally resets the database and storage.. there really isn't a reason it should not also reindex. I will address the upstream issue in the next PR.

Reply:
Fixed